src/Catel.Core/Catel.Core.Shared/Logging/CatelLog.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+
+namespace Catel.Logging
+{
+    using Interfaces;
```

Comment:
We never have .Interfaces namespaces, but this should be changed in the interface definition